bin/check-artifact.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+#
+# A simple check that our release .aar files are a reasonable size.
```

Comment:
I am not the most well versed in shell script, but isn't this script just checking that the required files _exist_? Or is there a size check that I missed?

Reply:
Woops, the comment there is wrong. We only check for the contents, not the size